llvm/test/DebugInfo/X86/is_stmt-at-block-start.ll

```diff
@@ -0,0 +1,62 @@
+;; Checks that when an instruction at the start of a BasicBlock has the same
+;; DebugLoc as the instruction at the end of the previous BasicBlock, we add
+;; is_stmt to the new line, to ensure that we still step on it if we arrive from
+;; a BasicBlock other than the immediately preceding one, unless all known
+;; predecessor BasicBlocks end with the same line.
+
```

Comment:
Convention is to put the RUN lines above this IIRC? Either way I endorse every test having an explanation of what it's testing.